include/unifex/when_all.hpp

```diff
@@ -33,221 +33,264 @@
 #include <variant>
 
 namespace unifex {
-
-namespace detail {
+namespace _when_all {
 
 template <
     std::size_t Index,
     template <std::size_t> class Receiver,
     typename... Senders>
-struct when_all_operation_tuple;
-
-template <std::size_t Index, template <std::size_t> class Receiver>
-struct when_all_operation_tuple<Index, Receiver> {
-  template <typename Parent>
-  explicit when_all_operation_tuple(Parent&) noexcept {}
-
-  void start() noexcept {}
+struct operation_tuple_ {
```

Comment:
I don't think you need to apply the ADL-isolation techniques to operation_tuple as that's just an internal type that is never passed to any ADL calls.

Reply:
Yeah, this was an overabundance of caution. I am inclined to leave it though.